### PR TITLE
New Mailer

### PR DESCRIPTION
**Pull request by aayush-05**
This PR contains the following changes:
--Responsive new mailer views
--Revamped Custom mail view admin panel

### CODE REVIEW

**Comment by aayush-05** [comment]
@nitin10s @tachyons please have a look at this.
PS: They are just too many to take screenshots of depicting all responsiveness. Please, redirect to
/rails/mailers -> previews for Mailer views
/custom_mails -> Admin panel for custom mails

File: app/assets/stylesheets/signup.scss [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
-.form-container {
-  margin-left: 40px;
-  margin-top: 30px;
-  max-width: 80%;
-}
+// .form-container {
+//   margin-left: 40px;
+//   margin-top: 30px;
+//   max-width: 80%;
+// }
```
Comment (tachyons) [inline app/assets/stylesheets/signup.scss:13]: ✂️

**Review by tachyons — APPROVED** [review]

File: app/assets/stylesheets/signup.scss (hunk 1 above)
Comment (aayush-05) [reply app/assets/stylesheets/signup.scss:13]: This stylesheet as a whole will be removed

**Comment by satu0king** [comment]
Excellent work

**Comment by nitin10s** [comment]
The CV logo appears on the left on all templates.

**Comment by aayush-05** [comment]
I made it in reference to how brand logo appears on Navbars (at the left) and other brand mails (see image). Let me know if I should change it to center? @nitin10s 
image

**Pull request merged**